vendor/browan/tbam100.js

```diff
@@ -1,6 +1,11 @@
 function decodeUplink(input) {
     var bytes = input.bytes;
 
+     //Check if temperature Board byte is present
```

Comment:
I think you should add these checks inside the the switch case instead, before the return. Then you can keep the `unknown FPort` error.